src/Blast/Bundle/UIBundle/Resources/views/Grid/Table/header.html.twig

```diff
@@ -0,0 +1,64 @@
+{% import _self as table %}
+
+<tr>
+    {% if grid.definition.actionGroups|length > 0 and 'bulk' in grid.definition.actionGroups|keys %}
+        <th class="center aligned">
+            <div class="ui fitted checkbox select all">
+                <input type="checkbox" autocomplete="off">
```

Comment:
Why not use blast_widget_field helper here ?

Reply:
Nice catch, I've missed that (for row and footer too)